packages/fe-container/src/components/top-bar/TopBar.tsx

```diff
@@ -25,20 +27,25 @@ import {UserContext} from '@contexts/User.context'
 export const TopBar: React.FC = () => {
   const configuration = useContext(ConfigurationContext)
   const user = useContext(UserContext)
+  const mustShowBurgerIcon = (configuration?.plugins || []).length > 0
 
   return (
     <div className='topBar_container'>
-      <BurgerIcon/>
+      {mustShowBurgerIcon && <BurgerIcon />}
       <img
-        alt={configuration?.theming?.logo.alt || 'Logo'}
+        alt={configuration.theming?.logo.alt || 'Logo'}
         className='logo'
         data-testid='company-logo'
-        src={configuration?.theming?.logo.url}
+        src={configuration.theming?.logo.url}
       />
-      <div className='topBar_userMenu'>
-        {
-          user.name && <UserMenu {...user}/>
-        }
+      <div className = 'topBar_rightSide'>
+        <HelpIcon/>
+        <Divider className='topBar_divider' type="vertical"/>
```

Comment:
the divider should be inserted only if user is logged! (see use.name && ... condition)